cornflow/commands/schemas.py

```diff
@@ -0,0 +1,34 @@
+def update_schemas_command(url, user, pwd, verbose):
+    import time
+
+    #
+    from cornflow_client.airflow.api import Airflow
+
+    from sqlalchemy.exc import IntegrityError
+    from ..models import DeployedDAG
+    from ..shared.utils import db
+
+    af_client = Airflow(url, user, pwd)
+    max_attempts = 20
+    attempts = 0
+    while not af_client.is_alive() and attempts < max_attempts:
+        attempts += 1
+        if verbose == 1:
+            print(f"Airflow is not reachable (attempt {attempts})")
+        time.sleep(15)
+
+    if not af_client.is_alive():
+        if verbose == 1:
+            print("Airflow is not reachable")
+        return False
+
+    response = af_client.update_schemas()
```

Comment:
I can't find the `update_schemas()` function anywhere in the cornflow_client repository, is it because you haven't created it yet?

Reply:
It is in a branch of cornflow_client (feature/list_dags)